### PR TITLE
Added latin_hypercube docs

### PR DESCRIPTION
**Pull request by ludoro**

### CODE REVIEW

File: docs/src/tuning_models.md [hunk 1]
```diff
@@ -173,6 +173,53 @@ The prior distributions used for sampling each hyperparameter can be
...
+latin_minimal = LatinHypercube(nGenerations=2,
+                               popSize= 120,
+                               nTournament = 2,
+                               pTournament = 0.3,
+                               interSampleWeight = 0.7,
+                               ae_power = 1
+                               periodic_ae = True,
+                               rng = 1234)
+```
+Where nTournament is the number of samples selected for
```
Comment (ablaom) [inline docs/src/tuning_models.md:212]: "samples" ->"`samples`"
"tournament" -> "`tournament`" etc

File: docs/src/tuning_models.md [hunk 2]
```diff
@@ -173,6 +173,53 @@ The prior distributions used for sampling each hyperparameter can be
...
+                               interSampleWeight = 0.7,
+                               ae_power = 1
+                               periodic_ae = True,
+                               rng = 1234)
+```
+Where nTournament is the number of samples selected for 
+tournament, pTournament is the probability for 
+tournament selection, interSampleWeight is the sampling 
+weight, ae_power is the norm used in the Audze-Eglais 
+function and periodic_ae is a boolean which allos the
```
Comment (ablaom) [inline docs/src/tuning_models.md:216]: allos -> allows

File: docs/src/tuning_models.md [hunk 3]
```diff
@@ -173,6 +173,53 @@ The prior distributions used for sampling each hyperparameter can be
...
+We can also tune the hyperparameters using `LatinHypercube`. This 
+method instantiates a grid-based hyperparameter tuning strategy 
+by using a genetic based optimization algorithm based on the 
+inverse of the Audze-Eglais function, using the library 
+`LatinHypercubeSampling.jl`.
+Let's instatiate the tuning strategy:
+```@repl goof
+latin_minimal = LatinHypercube(nGenerations=2,popSize= 120)
+```
+Where nGenerations is the number of generations to run the
```
Comment (ablaom) [inline docs/src/tuning_models.md:187]: I'm a bit confused by the definition of `nGenerations` and the other terminology. Presumably, this has nothing to do with the iteration parameter `n` specified in the `TunedModel` construction, but the way it is presented here makes it sound like it *is* the iteration parameter. Also, is it really fair to call this genetic *optimisation*? Aren't we just spitting out the entire set of models to be evaluated in one hit? When I think of [genetic optimisation](https://towardsdatascience.com/genetic-algorithm-to-optimize-machine-learning-hyperparameters-72bd6e2596fc) I think of a sequential algorithm in which the choice of new models to be evaluated at each step depends on the outcomes of previous ones.

**Review by ablaom — COMMENTED** [review]
Thanks @ludoro .

File: docs/src/tuning_models.md (hunk 3 above)
Comment (ablaom) [reply docs/src/tuning_models.md:187]: I'm probably just ignorant about the algorithm, but others are likely to be confused.

Comment (ludoro) [reply docs/src/tuning_models.md]: I see what you mean, after reading again what I have written I see where the confusion comes from. The inner working of the library is actually based on a genetic algorithm, and all the parameters refer to this in one way or the other. So I just made the distinction clearer, let me know if that's better now.

File: docs/src/tuning_models.md [hunk 4]
```diff
@@ -184,9 +184,15 @@ Let's instatiate the tuning strategy:
 ```@repl goof
 latin_minimal = LatinHypercube(nGenerations=2,popSize= 120)
 ```
-Where nGenerations is the number of generations to run the 
-optimisation for and popSize is the population size in 
-the genetic algorithm. 
+Where `nGenerations` is the number of generations to run the 
+optimisation for and `popSize` is the population size in 
+the genetic algorithm. Pay attention that these parameters
+have nothing to do with the ones specified in the `TunedModel`
```
Comment (ablaom) [inline docs/src/tuning_models.md:190]: Much better.  Instead of "ones specified in" with "the iteration parameter `n` in".

**Review by ablaom — APPROVED** [review]
Just the one last finicky suggestion above.

**Review by ablaom — APPROVED** [review]
Thanks.

**Pull request merged**